data/es/2020/hispano/speaker.yml

```diff
@@ -0,0 +1,43 @@
+items:
+  - name: "Alex Soto"
+    title: "Director of Developer Experience at Red Hat"
+    img: "asoto.jpg"
+  - name: "Cesar Hernandez"
+    title: "Senior Software Engineer en Tomitribe"
+    img: "chernandez.jpg"    
+  - name: "Fabio Turizo"
+    title: "Software Engineer at Payara"
+    img: "fturizo.jpg"
+  - name: "Jorge Cajas "
+    title: "Software Engineer at MangoChango"
+    img: "jcajas.jpg"
+  - name: "Jorge Vargas"
+    title: "CTO Weex"
+    img: "jvargas.jpg"
+  - name: "Marco Toscano"
+    title: "Software Architect and Developer"
+    img: "mtoscano.jpg"    
+  - name: "Mercedes Wyss"
+    title: "Software Engineer"
+    img: "mwyss.jpg"
+  - name: "Sebastian Daschner"
+    title: "Lead Java Developer Advocate at IBM"
+    img: "sdaschner.jpg"
+  - name: "Isaac Ruiz"
+    title: "Software Architect"
+    img: "iruiz.png"
```

Comment:
Did you mean for this to be "iruiz.jpg"?

Reply:
no, it is png, it is a generic image, the speaker has not sent his profile picture.

Reply:
Ready, i removed the picture .png and update jpg.